apps/src/componentLibrary/dropdown/simpleDropdown/simpleDropdown.module.scss

```diff
@@ -1,11 +1,15 @@
-@import "color";
-@import "@cdo/apps/componentLibrary/common/styles/mixins";
-@import "@cdo/apps/componentLibrary/typography/typography.module";
+@import 'color';
+@import '@cdo/apps/componentLibrary/common/styles/mixins';
+@import '@cdo/apps/componentLibrary/typography/typography.module';
 
 // Dropdown common styles
 .dropdownContainer {
   position: relative;
-  display: inline-block;
+  display: inline-flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 0.125rem;
+  min-width: 18.75rem;
```

Comment:
Upon discussion with design team, please remove this from PR for now and add minWidth with help of custom `className` please